README.md

```diff
@@ -1,2 +1,62 @@
 # github-actions
 A collection of reusable and composite Github Actions
+
+## gradle-check-tos
+A reusable action to check that Gradle Terms of Service have been approved.
+
+The action succeeds if the PR contributor(s) are recorded in the signature file, fails otherwise.
+
+**Dependencies**:
+
+- [cla-assistant-lite](https://github.com/marketplace/actions/cla-assistant-lite)
+
+**Event Trigger**:
+- `pull_request_target`
+- `issue-comment`
+
+**Action inputs**:
+
+| Name                                   | Description                                                                | Default                                                                                                                                                    |
+|----------------------------------------|----------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tos-location`                         | Terms Of Service location (URL or relative file)                           |                                                                                                                                                            |
```

Comment:
How does this work if it's a relative file? Does the user get a link to open the file in GitHub?
If so, this could avoid our need to wait until the OSS TOS are published: we could simply commit the TOS to the repository (with permission from legal, I guess).

Reply:
[You can't use a relative path in a PR comment](https://github.com/github/markup/issues/576). The base path being the PR URL.
Absolute path is the way to link to files in PR comments afaik.

This can definitely be a file inside the repository.

Reply:
Got it. I think this is fine.